src/components/textfieldControl/js/TextfieldStepper.js

```diff
@@ -74,18 +79,13 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
                             listener: "fluid.textfieldStepper.bindKeyEvent",
                             // down arrow === 40
                             args: ["{that}.container", "keydown", 40, "{textfieldStepper}.decrease"]
-                        },
-                        "onCreate.addRole": {
-                            "this": "{that}.container",
-                            method: "attr",
-                            args: ["role", "spinbutton"]
                         }
                     },
                     modelListeners: {
                         "value": {
                             "this": "{that}.container",
                             method: "attr",
-                            args: ["aria-valuenow", "{change}.value"]
+                            args: ["value", "{change}.value"]
```

Comment:
It seems from the MDN docs that this attribute controls a default rather than a current value - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#value - what's the accessibility impact of removing this listener?

Reply:
Sorry, I should more properly say, "replacing this listener with one that changes the actual control value", via some means like $.val() or the direct "value" property, etc.

Reply:
This should already be handled by the textfield component. https://github.com/fluid-project/infusion/blob/75c3c7e12a96f0ed168e8e2c9691ce5b061309c3/src/components/textfieldControl/js/Textfield.js#L44-L50

Reply:
It appears that this listener can indeed be removed as it's handled in the textfield component. Resolved in https://github.com/fluid-project/infusion/pull/1049/commits/fe7b771984c45608edce74a6485453ea5cff0b5d.